emails/templates/ng_alert_notification.txt

```diff
@@ -1,38 +1,60 @@
-[[HiddenSubject .Subject "[[.Title]]"]]
-
-[[.Title]]
-----------------
-
-[[ .Alerts | len ]] alert[[ if gt (len .Alerts) 1 ]]s[[ end ]] for
-[[ range .GroupLabels.SortedPairs ]]
-[[ .Name ]] = [[ .Value ]]
-[[ end ]]
-[[ if gt (len .Alerts.Firing) 0 ]]([[ .Alerts.Firing | len ]]) Firing[[ end ]]
-[[ range .Alerts.Firing ]]
-Labels:
-[[ range .Labels.SortedPairs ]]
-[[ .Name ]] = [[ .Value ]]
-[[ end ]]
-[[ if gt (len .Annotations) 0 ]]
-Annotations:
-[[ end ]]
-[[ range .Annotations.SortedPairs ]]
-[[ .Name ]] = [[ .Value ]]
-[[ end ]]
-[[ end ]][[ if gt (len .Alerts.Resolved) 0 ]]([[ .Alerts.Resolved | len ]]) Resolved[[ end ]]
-[[ range .Alerts.Resolved ]]
-Labels:
-[[ range .Labels.SortedPairs ]]
-[[ .Name ]] = [[ .Value ]]
-[[ end ]]
-[[ if gt (len .Annotations) 0 ]]
-Annotations:
-[[ end ]]
-[[ range .Annotations.SortedPairs ]]
-[[ .Name ]] = [[ .Value ]]
-[[ end ]]
-[[ end ]]View your Alert rule:
-[[.RuleUrl]]
-
-Go to the Alerts page:
-[[.AlertPageUrl]]
+[[- define "__default_message" -]]
+You have [[ len .Alerts.Firing ]] firing alert(s), and [[ len .Alerts.Resolved ]] resolved alert(s) for [[ .GroupLabels ]]
+
+[[ if .Alerts.Firing -]]
+([[ len .Alerts.Firing ]]) FIRING
+-----------
+[[ template "__default_alerts_summarize" .Alerts.Firing ]]
+[[- end ]]
+
+[[ if .Alerts.Resolved -]]
+([[ len .Alerts.Resolved ]]) RESOLVED
+-------------
+[[ template "__default_alerts_summarize" .Alerts.Resolved ]]
+[[- end ]]
+Go to the Alerts page: [[ .AlertPageUrl ]]
+[[- end -]]
+
+[[- define "__default_alerts_summarize" -]]
+[[- range . ]]
+[[ template "__default_alert_labels" . ]]
+[[ template "__default_alert_annotations" . ]]
+[[- end ]]
+[[- end -]]
+
+[[- define "__default_alert_labels" -]]
+Labels: [[ .Labels.SortedPairs ]]
+[[- end -]]
+
+[[- define "__default_alert_annotations" -]]
+[[- $annotations := .Annotations -]]
+[[- if index $annotations "summary" -]]
+Summary: [[ index .Annotations "summary" ]]
+[[- $annotations = $annotations.Remove (toStrings (list "summary")) ]]
+[[ end -]]
+[[- if index $annotations "description" -]]
+Description: [[ index $annotations "description" ]]
+[[- $annotations = $annotations.Remove (toStrings (list "description")) ]]
+[[ end -]]
+[[- if index $annotations "runbook_url" -]]
+Runbook: [[ index $annotations "runbook_url" ]]
+[[- $annotations = $annotations.Remove (toStrings (list "runbook_url")) ]]
+[[ end -]]
+[[- if $annotations -]]
+Annotations: [[ template "__default_annotations" $annotations ]]
+[[ end -]]
+[[- end -]]
+
+[[- define "__default_annotations" -]]
+[[ range .SortedPairs ]]
+    [[ .Name ]] = [[ .Value ]]
+[[- end ]]
+[[- end -]]
+
+[[- if .Message -]]
+    [[ .Message ]]
+[[ else -]]
+    [[ template "__default_message" . ]]
+[[- end ]]
+
+Sent by Grafana v[[.BuildVersion]] (c) [[now | date "2006"]] Grafana Labs
```

Comment:
The footer is added by the `make` command from `emails/templates/layout/default.txt` so no need to include it here.